smartapps/curb/curb-energy-monitor.src/curb-energy-monitor.groovy

```diff
@@ -16,7 +16,7 @@
 include 'asynchttp_v1'
 
 definition(
-    name: "CURB Energy Monitor",
+    name: "CURB Connect",
```

Comment:
name changes for groovy smartapps are too painful at point with the file name and also associated with the name. For now lets revert this back